test/policies/stripe_account_policy_test.exs

```diff
@@ -0,0 +1,61 @@
+defmodule CodeCorps.StripeAccountPolicyTest do
+  use CodeCorps.PolicyCase
+
+  import CodeCorps.StripeAccount
+  import CodeCorps.StripeAccountPolicy, only: [show?: 2]
+
+  alias CodeCorps.StripeAccount
+
+  describe "show?" do
+    test "returns true when user is an admin" do
+      user = build(:user, admin: true)
+      stripe_account = insert(:stripe_account)
+
+      assert show?(user, stripe_account)
+    end
+
+    test "returns true when user is owner of organization" do
+      user = insert(:user)
+      organization = insert(:organization)
+      insert(:organization_membership, role: "owner", member: user, organization: organization)
+
+      stripe_account = insert(:stripe_account, organization: organization)
+
+      assert show?(user, stripe_account)
+    end
+
+    test "returns false when user is admin of organization" do
+      user = insert(:user)
+      stripe_account = insert(:stripe_account)
+
+      insert(:organization_membership, role: "admin", member: user)
+
+      refute show?(user, stripe_account)
+    end
```

Comment:
In the same way you'fe fixed the "returns true when user is owner of organization" test, you should fix all the other tests. 

In all of them, a membership is being created for the user in an organization that is unrelated to the stripe account organization. Really, while the tests say "returns false when user is X of organization", what they're all testing right now is "returns false when when user is not a member of organization".